packages/plugins/typescript/typescript/tests/typescript.spec.ts

```diff
@@ -1997,6 +2009,9 @@ describe('TypeScript', () => {
           MyScalar: { input: MyScalar['input']; output: MyScalar['output']; }
           MyOtherScalar: { input: MyOtherScalar['input']; output: MyOtherScalar['output']; }
           MyAliasedScalar: { input: AliasedScalar['input']; output: AliasedScalar['output']; }
+          OrgScalar: { input: OrgScalar['input']; output: OrgScalar['output']; }
+          OrgOtherScalar: { input: OrgOtherScalar['input']; output: OrgOtherScalar['output']; }
+          OrgAliasedScalar: { input: OrgAliasedScalar['input']; output: OrgAliasedScalar['output']; }
```

Comment:
I think it would have a much better backward compatibility if we don't require mapped scalar from external sources to be an input/output object

Will push a commit to fix this.

Reply:
Fixed in https://github.com/dotansimha/graphql-code-generator/pull/9418/commits/2665abf09ccd813dc994d824d015845315d203ff

Reply:
You've already done it, but agreed, this last commit is quite nice. Means the only code change we _had_ to do in our (internal) project was stop using Scalars[''] or add the ['input'] / ['output'] type.